lib/collection.js

```diff
@@ -1313,7 +1313,7 @@ var save = function(self, doc, options, callback) {
  * @param {boolean} [options.tailable=false] Specify if the cursor is tailable.
  * @param {number} [options.batchSize=0] Set the batchSize for the getMoreCommand when iterating over the query results.
  * @param {boolean} [options.returnKey=false] Only return the index key.
- * @param {number} [options.maxScan=null] Limit the number of items to scan.
+ * @param {number} [options.maxScan=null] DEPRECATED: Limit the number of items to scan.
```

Comment:
we should also warn about deprecation in all `find` methods that support the `maxScan` option